expirationd.lua

```diff
@@ -962,8 +1004,15 @@ local function expirationd_run_task(name, space_id, is_tuple_expired, options)
         end
     end
 
-    -- check valid of iterator_type and start key
-    task.index:pairs( task.start_key(), { iterator = task.iterator_type })
+    local ok, err = check_space_and_index_exist(task)
+    if ok then
+        local ok, err = check_space_and_index(task)
+        if not ok then
+            error(err)
+        end
+    else
+        log.info("schedule task " .. name .. ", reason: " .. err)
```

Comment:
Did you mean 'postpone'? 'Schedule' sounds for me as something about completely valid/expected case, but here we should warn the user. BTW, shouldn't it be a on the warnings log level?

Reply:
Good ideas. Now the message in the log looks like this:
`W> expiration: postpone a task clean_all, reason: Index with name non_existing_name does not exist`